nitrocli/src/pinentry.rs

```diff
@@ -21,7 +21,45 @@ use error::Error;
 use std::process;
 
 
-const CACHE_ID: &str = "nitrocli:user";
+/// PIN type requested from pinentry.  The available PIN types correspond to the PIN types used by
```

Comment:
I believe it is good practice to make the first sentence a summary and to separate it from the body with an empty line. E.g., https://github.com/rust-lang/rust/blob/master/src/libcore/str/mod.rs#L81